fe1-web/model/objects/Chirp.ts

```diff
@@ -0,0 +1,87 @@
+import { Timestamp } from './Timestamp';
+import { Hash } from './Hash';
+
+/**
+ * Object to represent a Chirp.
+ */
+
+export interface ChirpState {
+  id: string;
+  sender: string;
+  text: string;
+  time: number;
+  likes: number;
+  parentId: string;
+}
+
+export class Chirp {
+  public readonly id: Hash;
+
+  // The sender's public key
+  public readonly sender: string;
+
+  // The text of the chirp
+  public readonly text: string;
+
+  // The time where the chirp was posted
+  public readonly time: Timestamp;
+
+  // The number of likes
+  public readonly likes: number;
```

Comment:
I seem to remember that the social media project has this overall definition of reactions, which could be likes (👍) or any other Unicode character (e.g., 🤮).

Has this been simplified (on both front-ends) to just handling likes ? Or should we support more than just a number here ?

Reply:
We decided to simplify it to support only likes for the midterm presentation and add the dislike later.

Reply:
Yes, for now we kept only likes, as it seems easier to implement.

Reply:
You're right, but actually I don't think we're going to support any reaction other than likes. We might need to talk about it next social media meeting though.